### PR TITLE
Add deviation towards lateral side template

### LINKED ISSUE #838: Deviation towards the lateral side of anatomical entity pattern template
* Examples:
    - [HP:0001822	Hallux valgus](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0001822)
    - [HP:0003049	Ulnar deviation of the wrist](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0003049)
    - [HP:0100500	Fibular deviation of toes](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0100500)

### PR DESCRIPTION
**Pull request by rays22**
This commit intends to
add the `deviationTowardsLateralSideOfAnatomicalEntity.yaml` DOS-DP  pattern draft for community review.
If applied, this commit will fix #838.

### CODE REVIEW

File: src/patterns/dosdp-dev/deviationTowardsLateralSideOfAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - anatomical_entity
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "%s lateral deviation"
+    vars:
+      - anatomical_entity
+
+def:
+  text: "A changed direction of position towards the side of %s."
```
Comment (rays22) [inline src/patterns/dosdp-dev/deviationTowardsLateralSideOfAnatomicalEntity.yaml:44]: This looks OK.

**Comment by sbello** [comment]
this looks okay ORCID: orcid.org/0000-0003-4606-0597

File: src/patterns/dosdp-dev/deviationTowardsLateralSideOfAnatomicalEntity.yaml (hunk 1 above)
Comment (rays22) [reply src/patterns/dosdp-dev/deviationTowardsLateralSideOfAnatomicalEntity.yaml:44]: A changed direction of position or shift towards the side of %s.

**Comment by ybradford** [comment]
this looks good to me Orcid id 0000-0002-9900-7880

**Pull request merged**